Automattic/wp-calypso
# Revert "Reader: Reimplement keyboard shortcuts"

Reverts Automattic/wp-calypso#60812, which seems like it may be the cause of https://github.com/Automattic/wp-calypso/issues/61063.

### Testing instructions

Load a post in Reader that has comments enabled, and in the comment input try typing text that includes the letter `l`. This should not like the post. Example post:

https://wordpress.com/read/feeds/40474296/posts/2988003678
